common/app/implicits/Requests.scala

```diff
@@ -108,8 +108,11 @@ trait Requests {
 
     // dotcom-rendering (DCR) parameters
     lazy val forceDCROff: Boolean = r.getQueryString("dcr").contains("false")
-    lazy val forceDCR: Boolean =
```

Comment:
Not the most familiar with Scala syntax, is there a reason this needs to be wrapped in `{}`?

Reply:
when there's only one line of code you can choose to avoid the {}, so it's only a matter of preference, but I should have made them both the same for consistency! Which do you prefer?

Reply:
For 1 line I prefer not having the `{}` ! Personal preference though!